applications/contents/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class Content(models.Model):
+    key = models.CharField(max_length=20, default='', unique=True)
+    title = models.CharField(max_length=100)
+    body = models.CharField(max_length=1000)
```

Comment:
If this allows a lot of text then you should use TextField
https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.TextField